news/2020-04-01-How-to-configure-wildfly-s2i-by-cli-operations.adoc

```diff
@@ -0,0 +1,446 @@
+...
+Successfully pushed image-registry.openshift-image-registry.svc:5000/wildfly-demo/wildfly-app@sha256:6057c3bbc0a9071b102b4d0404f9592edebb0ef7c4dfbca9b00e50a2a117adcd
+Push successful
+----
+
+We can see in the log how the image source mount named 'extensions' was processed, the value of our injected dir, which is a directory in the filesystem of the image being assembled and a trace that tell us the server was configured by the embedded server.
+
+Let us now check test the application exposing the application to the outside world and fetching some information:
+
+[source, bash]
+----
+$ oc expose svc/wildfly-app --name wildfly-app
+route.route.openshift.io/wildfly-app exposed
+
+$ curl http://$(oc get routes/wildfly-app --template={{.spec.host}})/jaxrs-postgresql-demo/api/tasks
+[{"id":1,"title":"This is the task-1"},{"id":2,"title":"This is the task-2"},{"id":3,"title":"This is the task-3"},{"id":4,"title":"This is the task-4"},{"id":5,"title":"This is the task-5"}]
+----
+
+We can also open a remote connection and inspect the relevant data source configuration:
+
+The datasources subsystem configuration is the following:
+
+[source, bash]
+----
+$ oc rsh wildfly-app-1-dvnv6
+sh-4.2$ cat /opt/wildfly/standalone/configuration/standalone.xml
+----
+
+[source, xml]
+----
+<subsystem xmlns="urn:jboss:domain:datasources:5.0">
+    <datasources>
+        <datasource jndi-name="java:/jboss/datasources/PostgreSQLDS" pool-name="PostgreSQLPool">
+            <connection-url>jdbc:postgresql://database-server:5432/demodb</connection-url>
+            <driver>postgresql</driver>
+            <security>
+                <user-name>postgre</user-name>
+                <password>admin</password>
+            </security>
+        </datasource>
+        <drivers>
+            <driver name="postgresql" module="org.postgresql.jdbc">
+                <xa-datasource-class>org.postgresql.xa.PGXADataSource</xa-datasource-class>
+            </driver>
+        </drivers>
+    </datasources>
+</subsystem>
+----
+
+Now clean up the wildfly-app keeping the PostgreSQL server running, we will use it for the next example:
+
+[source, bash]
+----
+$ oc delete all -l app=wildfly-app
+pod "wildfly-app-1-dvnv6" deleted
+replicationcontroller "wildfly-app-1" deleted
+service "wildfly-app" deleted
+deploymentconfig.apps.openshift.io "wildfly-app" deleted
+buildconfig.build.openshift.io "wildfly-app" deleted
+build.build.openshift.io "wildfly-app-1" deleted
+imagestream.image.openshift.io "wildfly-app" deleted
+----
+
+== Using the extension mechanism to configure the Server
+
+The extension mechanism allows the execution of arbitrary bash scripts before and after the server is configured by using environment variables. An interesting use case could be you want to tweak the server configuration after being configured by the environment variables, for example, there is a specific configuration it is not exposed directly by an environment variable.
```

Comment:
s/after being/after it has been/g

I think ^^^ is your meaning; if not please ignore.

Reply:
s/it is not/that is not/g